inc/extras.php

```diff
@@ -69,3 +69,220 @@ function llorix_one_lite_render_title() {
 	}
 	add_action( 'wp_head', 'llorix_one_lite_render_title' );
 endif;
+
+
+if(!function_exists('llorix_one_lite_pll')){
+	/**
+	 * Filter to apply translations if polylang plugin is installed.
+	 * @param string $input Input to translate.
+	 *
+	 * @return string
+	 */
+	function llorix_one_lite_pll($input){
+		if(function_exists('pll__')){
+			return pll__($input);
+		}
+		return $input;
+	}
+	add_filter('llorix_one_lite_language_filter','llorix_one_lite_pll');
+}
+
+
+if(!function_exists('llorix_one_companion_register_string')){
+	/**
+	 * Function to register repeater content in polylang.
+	 *
+	 * @param string $input String to register.
+	 * @param string $context Field name.
+	 */
+	function llorix_one_companion_register_string($input,$context){
+		$json_decoded = json_decode($input,true);
+		foreach($json_decoded as $index => $value){
+			foreach ($value as $key => $string){
+				if($key !== 'id' && $string !== 'undefined'){
+					$text = false;
+					if($key === 'text'){
+						$text = true;
+					}
+					pll_register_string('llorix one '.$context, $string, $text);
```

Comment:
http://prnt.sc/cvd81g when polylang is not installed I think, you need to check if the pll_register_string exists

Actually I tested again, and with polylang installed, I get the same fatal error